.github/workflows/run_android.yml

```diff
@@ -32,12 +32,29 @@ jobs:
       run: |
         echo "y" | sudo ${ANDROID_HOME}/tools/bin/sdkmanager --install "ndk;${ANDROID_NDK_VERSION}" --sdk_root=${ANDROID_SDK_ROOT} > /dev/null
     # build for all ABIs
+    - name: Compile libusb
+      run: |
+        git clone --depth 1 --branch android https://github.com/xloem/libusb.git
```

Comment:
I saw your PR in upstream, there is very good feedback. How do you think when they will merge it?

Reply:
Right now it looks like they're looking for successful test reports to move things forward, and that more of those will happen if I add a requested feature.  I was hoping to get the brainflow test app going with the changes and kind of meet two goals with one task, but I haven't gotten to that yet.